app/pages/home-for-user/circle-ribbon.jsx

```diff
@@ -168,6 +170,8 @@ const CircleRibbon = React.createClass({
       tooltipPosition = this.getTooltipPoint(hoveredArc, 50);
     }
 
+    const avatar = this.props.user.avatar_src ? this.props.user.avatar_src : '/assets/simple-avatar.png';
```

Comment:
Can this just be `const avatar = this.props.user.avatar_src` since the default is always set to `'/assets/simple-avatar.png'`?